src/extension/iSamples_resultList.js

```diff
@@ -109,40 +56,43 @@ class ResultList extends React.Component {
 
     if (bbox) { delete bbox['error'] }
 
+    // view style
+    const showView = (targetView) => ({ display: view['facet'] === targetView ? "block" : "none" });
+
     return (
       <ButGroup
         bootstrapCss={bootstrapCss}
-        switchFormat={this.switchFormat.bind(this)}
-        active={this.state.facet}
-        children={
```

Comment:
The elements between two JSX tags would automatically be the children. e.g. `<component>{test}</component>`.  `test` would be the children property of `component` props